op-acceptance-tests/justfile

```
@@ -98,8 +98,8 @@ acceptance-test devnet="" gate="base":
             "--validators" "./acceptance-tests.yaml"
             "--exclude-gates" "flake-shake"
             "--allow-skips"
-            "--timeout" "90m"
-            "--default-timeout" "30m"
+            "--timeout" "60m"
+            "--default-timeout" "20m"
```

```suggestion
            "--default-timeout" "30m"
```

Suggest not reducing this as it's already below the circle ci limit.

---

Removed this in the end (ineffective as 'timeout' overrides it)